README.md

```diff
@@ -91,6 +91,10 @@ For releasing a new stable version from main branch, go to
 [Actions](https://github.com/getsentry/sentry-kafka-schemas/actions) and
 trigger a new job for the `Release` workflow.
 
+After releasing a new version, you should immediately bump Sentry, Snuba and
```

Comment:
Is there an order to how these should happen? Or doesn't matter?

Reply:
It shouldn't matter